src/docs/admin/references/NodeSourceInfrastructureReference.adoc

```diff
@@ -735,21 +735,38 @@ You can opt to place this file in `$PROACTIVE_HOME/config/authentication/azure.c
 - *osType:* A choice between _linux_ and _windows_. Default value is _linux_.
-- *Image:* You can either provide a custom Azure image, or select one of the available standard ones. To provide a custom image use the format _{resourceGroupName}:{imageName}_. If you prefer to use a generic image, you can choose from the following lists (provided by Azure API):
+- *Image:* There are four possible ways to specify the source image for your Scale Set VMs: _known images_, Azure Marketplace, Shared Image Gallery and Custom images. We describe each of them below.
+* *Syntax summary*
+** Known image: `MY_KNOWN_IMAGE`
+** Marketplace: `marketplace#publisher:offer:sku[:version]`
+** Shared Image Gallery: `gallery#resourceGroup:gallery:image[:version]`
+** Custom image: `custom#resourceGroup:customImage`
+* *_Known image._* Refers to commonly used images as defined in Azure's SDK. Accepted values are simple strings, as listed in the following table:
+
 +
 |===
 |https://github.com/Azure/azure-libraries-for-java/blob/master/azure-mgmt-compute/src/main/java/com/microsoft/azure/management/compute/KnownLinuxVirtualMachineImage.java[Linux] |https://github.com/Azure/azure-libraries-for-java/blob/master/azure-mgmt-compute/src/main/java/com/microsoft/azure/management/compute/KnownWindowsVirtualMachineImage.java[Windows]
 
-a|- UBUNTU_SERVER_14_04_LTS
-  - UBUNTU_SERVER_16_04_LTS
-  - DEBIAN_8 +*+ _default value_ +*+
-  - CENTOS_7_2
+|UBUNTU_SERVER_14_04_LTS +
+  UBUNTU_SERVER_16_04_LTS +
+  DEBIAN_8 +*+ _default value_ +*+ +
+  CENTOS_7_2
 
-a|- WINDOWS_SERVER_2008_R2_SP1
-  - WINDOWS_SERVER_2012_DATACENTER
-  - WINDOWS_SERVER_2012_R2_DATACENTER
+|WINDOWS_SERVER_2008_R2_SP1 +
+  WINDOWS_SERVER_2012_DATACENTER +
+  WINDOWS_SERVER_2012_R2_DATACENTER
 |===
 
+* *_Azure Marketplace._* If you want to choose a particular image from Azure's Marketplace, use the prefix `marketplace#` plus the image's https://docs.microsoft.com/en-us/azure/virtual-machines/linux/cli-ps-findimage[URN] in the form `publisher:offer:sku[:version]`. Similarly to Azure CLI or PowerShell, ProActive allows you to specify a version of the image (with `:version`), or it will use the latest one if the version is omitted. Examples of valid inputs are: +
+`marketplace#OpenLogic:CentOS:7.5` +
+`marketplace#OpenLogic:CentOS:7.5:latest` +
+`marketplace#credativ:Debian:8:8.0.201901221`
+* *_Shared Image Gallery:_* The prefix `gallery#` allows you to use images from an existing Azure https://docs.microsoft.com/en-us/azure/virtual-machines/linux/shared-image-galleries[Shared Image Gallery]. Simply provide the name of your Resource Group, your Gallery and your Image as `resourceGroup:gallery:image`. Optionally, you can specify a version too (`:version`); if no version is given, the latest image version will be used by ProActive. For example: +
```

Comment:
I think that you should remove `Simply` in the sentence `Simply provide the name [...]`. I think that this usage doesn't belong to a professional documentation.

Reply:
thanks Mael, I'll fix it.